python/lsst/pipe/supertask/activator.py

```diff
@@ -350,15 +385,16 @@ def get_tasks(modules_only=False):
                     module_classes = [class_key for class_key in classes_map.keys() if
                                       classes_map[class_key].module == module_name]
                     for mod_cls in module_classes:
```

Comment:
if these are going through a list of class names I think I'd like `name` in the variable name. Otherwise it looks like we are looping over class objects.